### PR TITLE
Meeting 2024-11-20 minutes

### PR DESCRIPTION
**Pull request by sthagen**
- [x] pre-meeting draft
- [x] pre-meeting syncs (roster, agenda, ...)
- [x] post-meeting draft
- [x] ready for review

### CODE REVIEW

File: meeting-minutes/2024-11-20.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  - regrets from Stefan Hagen
+
+# 3. Discussion
+
+- [Meeting minutes August and September #50](https://github.com/oasis-tcs/openeox/issues/50)
+  - Stefan: Please provide the recordings to fix the roster
+  - Justin: Will see that the recordings will be sent to Stefan for extracting
+- [CISA proposed definition for "security support" as part of NCSIP Initiative 3.3.2 #51](https://github.com/oasis-tcs/openeox/issues/51)
+  - Justin: Motivates the issue and invites everyone to discuss the proposed definition in the issue
+  - Langley: We are not responding to security events or security incidents but not to security risks
```
Comment (lfrancke) [inline meeting-minutes/2024-11-20.md:101]: I assume one of these "not"s is wrong?

Comment (justmurphy) [reply meeting-minutes/2024-11-20.md:101]: @lfrancke good catch. As noted in #51, should read as:
"**Not** responding to security risk, but to security events/incidents."

Comment (justmurphy) [reply meeting-minutes/2024-11-20.md]: Resolved. Thanks for the catch @lfrancke

File: meeting-minutes/2024-11-20.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+# 3. Discussion
+
+- [Meeting minutes August and September #50](https://github.com/oasis-tcs/openeox/issues/50)
+  - Stefan: Please provide the recordings to fix the roster
+  - Justin: Will see that the recordings will be sent to Stefan for extracting
+- [CISA proposed definition for "security support" as part of NCSIP Initiative 3.3.2 #51](https://github.com/oasis-tcs/openeox/issues/51)
+  - Justin: Motivates the issue and invites everyone to discuss the proposed definition in the issue
+  - Langley: We are responding to security events or security incidents, but not to security risks 
+- [End-of-Software Maintenance #29](https://github.com/oasis-tcs/openeox/issues/29) 
+  - ... based onn last month's discussion for [Using "End" word in the lifecycle definitions #38](https://github.com/oasis-tcs/openeox/issues/38)
```
Comment (tschmidtb51) [inline meeting-minutes/2024-11-20.md:103]:
```suggestion
  - ... based on last month's discussion for [Using "End" word in the lifecycle definitions #38](https://github.com/oasis-tcs/openeox/issues/38)

File: meeting-minutes/2024-11-20.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  - Justin: Moves to close the issue [Using "End" word in the lifecycle definitions #38](https://github.com/oasis-tcs/openeox/issues/38)
+    - Przemyslaw: Seconds
+      - No discussion, all in favor, motion acarries
```
Comment (tschmidtb51) [inline meeting-minutes/2024-11-20.md:106]:
```suggestion
      - No discussion, all in favor, motion carries

File: meeting-minutes/2024-11-20.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  - Przemyslaw: Updated the proposal to focus (at least for now) on only the end of periods and to allow specification of scopes
+  - Langley: Coupling feature with security updates? Feature is evolution, while maintenance is conserving, no?
```
Comment (tschmidtb51) [inline meeting-minutes/2024-11-20.md:108]: IIRC, it was Jay that made the comment about the evolution but I might be wrong
```suggestion
  - Langley: Coupling feature with security updates? 
  - Jautau: Feature is evolution, while maintenance is conserving, no?

File: meeting-minutes/2024-11-20.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    - <https://github.com/oasis-tcs/openeox/tree/main/value-scenarios/contributor-scenarios>
+    - <https://github.com/oasis-tcs/openeox/tree/main/value-scenarios/general-scenarios>
+- Langley: Agrees that some commonality needs to be reached
+- Przemyslaw: Thinks we already discussed and agreed; in any case we should progress towards a "minimal viable product"-like set of definitions / schema
+- Justin: Place use cases discussion on agenda for December meeting
+- Thomas: Open to both sequences of steps; the schema can always follow. If we need discuss scenarios first, we should follow, but we should progress no matter in which order.
+- Feng: Emphasizes his priority for reaching a common ground to be able to place OpenEOX on the market place
+- Justin: Proposes to invite everysone to visit the submitted scenarios and for others to submit or modify missing or incomplete scenarios and then follow over the next couple of meetings to share the scenarios
+- Sonny: Maybe use a full day or even a face to face meeting with a white board to discuss?
+- Justin: We could add an extra meeting for that purpose and bring back the resulkts to the regular TC meetings
```
Comment (tschmidtb51) [inline meeting-minutes/2024-11-20.md:135]:
```suggestion
- Justin: We could add an extra meeting for that purpose and bring back the results to the regular TC meetings

**Review by tschmidtb51 — COMMENTED** [review]
@sthagen Thank you for providing the notes! Just some typos and a comment that I'm not 100% sure of (and therefore might be safely ignore if the 2 persons concerned do not complain).

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Review by santosomar — APPROVED** [review]
Thank you, @sthagen !

**Comment by santosomar** [comment]
Motion to approve meeting minutes: https://groups.oasis-open.org/discussion/motion-to-approve-openeox-november-2024-meeting-minutes

**Review by justmurphy — APPROVED** [review]
LGTM, motion passed.

**Pull request merged**